Resources/Prototypes/_EE/Catalog/uplink_catalog.yml

```
@@ -0,0 +1,40 @@
+    Telecrystal: 50
+  categories:
+  - UplinkFaction
+  conditions:
+  - !type:BuyerSpeciesCondition
+    whitelist: [ Human ]
+
+- type: listing
+  id: UplinkFactionSolAllianceAssetGroup
+  name: uplink-faction-san-asset-group-name
+  description: uplink-faction-san-asset-group-desc
+  icon: { sprite: /Textures/Structures/Storage/Crates/aluminiumcrate.rsi, state: icon }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)